test/framework/ssh/ssh.go

```diff
@@ -20,57 +19,39 @@ func Logf(format string, args ...interface{}) {
 }
 
 type App struct {
-	cmd     *exec.Cmd
-	stdin   bytes.Buffer
-	stdout  bytes.Buffer
-	stderr  bytes.Buffer
-	port    string
-	logFile *os.File
+	cmd    *exec.Cmd
+	stdin  bytes.Buffer
+	stdout bytes.Buffer
+	stderr bytes.Buffer
+	port   string
 }
 
 func NewApp(appName string, verbose bool, port string, envMap map[string]string, args []string) *App {
 	app := &App{
 		port: port,
 	}
-	var env []string
+	sshArgs := []string{
+		"-q", "-tt",
+		"-o", "StrictHostKeyChecking=no",
+		"-o", "UserKnownHostsFile=/dev/null",
+		"root@localhost", "-p", port,
+	}
 	for k, v := range envMap {
-		env = append(env, fmt.Sprintf("%s='%s'", k, utils.ShellEscape(v)))
+		sshArgs = append(sshArgs, fmt.Sprintf("%s='%s'", k, utils.ShellEscape(v)))
 	}
-	sshArgs := append(
-		[]string{
-			"-q", "-tt",
-			"-o", "StrictHostKeyChecking=no",
-			"-o", "UserKnownHostsFile=/dev/null",
-			"root@localhost", "-p", port,
-		}, env...,
-	)
 	sshArgs = append(sshArgs, args...)
 	app.cmd = exec.Command("ssh", sshArgs...)
 
-	inWriters := []io.Reader{&app.stdin}
 	outWriters := []io.Writer{&app.stdout}
 	errWriters := []io.Writer{&app.stderr}
 
-	logDirName := "/tmp/e2e"
-	err := os.MkdirAll(logDirName, os.ModePerm)
-	if err != nil {
-		panic(errors.Wrap(err, "could not create /tmp/e2e"))
-	}
-	logFileName := logDirName + "/" + appName
-	f, err := os.OpenFile(logFileName, os.O_RDWR|os.O_APPEND|os.O_CREATE, 0660)
-	if err != nil {
-		panic(errors.Wrap(err, "could not create "+logFileName))
-	}
-	outWriters = append(outWriters, f)
-	errWriters = append(errWriters, f)
```

Comment:
why? :( I use it all the time. I know it's not ideal, because the logs are not appended after the test fails, but Is there a better way to inspect logs of universal containers now?

Reply:
I had a lot of problems with file descriptors leaking

Reply:
Is the verbose option not enough?

Reply:
the use case is that I want to inspect logs of a specific container. `docker ps` gives me nothing. Mixed logs of all containers in the full log (+ parallelization) make this impossible.